TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/managers/APIRequestManager.java

```diff
@@ -403,6 +403,14 @@ public void onPluginMessage(AddressedPluginMessage message) {
                             proxy.executeCommand(value);
                             break;
                         }
+                        case P_SEND_PLAYER:{
+                            String playerUUID = data.getString("playerUUID");
+                            String targetServer = data.getString("targetServer");
+                            validateNotNull(playerUUID, "UUID");
+                            validateNotNull(targetServer, "targetServer");
+                            proxy.sendPlayer(playerUUID, TimoCloudCore.getInstance().getInstanceManager().getServerByIdentifier(targetServer));
```

Comment:
We can check if the player is (still) on the given proxy and then set the API response to `true`, otherwise `false`